selenium/pageObject_model/PageSitePastebinResults.js

```diff
@@ -0,0 +1,92 @@
+import pkg from 'selenium-webdriver';
+const { Builder, Browser, By, until, wait } = pkg;
+import { HomePageSitePastebin } from './HomePageSitePastebin.js';
+
+export class PageSitePastebinResults extends HomePageSitePastebin {
+    constructor(driver) {
+        super(driver);
+        this.savedTitleXpath = '//div[@class="info-top"]/h1';
+        this.savedTextAreaXpath = '//div[@class="source"]/ol';
+        this.savedSyntaxXpath = '//div[@class="btn -small h_800"]';
+        this.savedExpirationXpath = '//div[@class="expire"]';
+        this.syntaxBashConfigXpath =
+            '((//*[@class="li1"])[1]/*[@class="de1"]/span)[1]';
+        this.syntaxBashResetXpath =
+            '((//*[@class="li1"])[2]/*[@class="de1"]/span)[1]';
+        this.syntaxBashCommitXpath =
+            '((//*[@class="li1"])[2]/*[@class="de1"]/span)[3]';
+        this.syntaxBashThreeXpath =
+            '((//*[@class="li1"])[2]/*[@class="de1"]/span)[5]';
+        this.syntaxBashPushXpath =
+            '((//*[@class="li1"])[3]/*[@class="de1"]/span)[1]';
+        this.syntaxBashGlobalXpath =
+            '(((//*[@class="li1"])[1]/*[@class="de1"])[1]/span)[2]';
+        this.syntaxBashMXpath =
+            '((//*[@class="li1"])[2]/*[@class="de1"]/span)[7]';
+        this.syntaxBashForceXpath =
+            '((//*[@class="li1"])[3]/*[@class="de1"]/span)[2]';
+        this.syntaxBashTextNameXpath =
+            '(((//*[@class="li1"])[1]/*[@class="de1"])[1]/span)[3]';
+        this.syntaxBashTextCommitXpath =
+            '((//*[@class="li1"])[2]/*[@class="de1"]/span)[8]';
```

Comment:
Please investigate how to ridd off these locator style (using [n]).

Reply:
done